app/assets/javascripts/image.js

```diff
@@ -0,0 +1,47 @@
+function loadImages() {
+  var item = $('#image_uuid');
+  var url = item.data('url');
+  var selected = item.data('selected');
+
+  item.indicator_show();
+  foreman.tools.showSpinner();
+  item.attr("disabled", true);
+
+  $.ajax({
+    type: 'get',
+    url: url,
+    complete: function() {
+      foreman.tools.hideSpinner();
```

Comment:
Unless there is a very good reason to have it under app/, all new code should be placed under webpack/ and written in es2015 syntax as modules.